recipes/sundials/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "sundials" %}
+{% set version = "2.6.2" %}
+{% set sha256 = "d8ed0151509dd2b0f317b318a4175f8b95a174340fc3080b8c20617da8aa4d2f" %}
+{% set md5 = "3deeb0ede9f514184c6bd83ecab77d95" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://pkgs.fedoraproject.org/repo/pkgs/sundials/sundials-2.6.2.tar.gz/{{ md5 }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - blas_{{ variant }}  # [not win]
+
+requirements:
+  build:
+    - cmake
+    - blas 1.1 {{ variant }}  # [not win]
+    - openblas 0.2.18*  # [not win]
+    - python
```

Comment:
It's just a config thing in that we actually make 3 different builds for Windows due to compiler requirements. Having the features in the build matrix ensures that the correct version is chosen for a given Python environment. So leave it as is and just copy those lines from the vlfeat recipe.